docs/source/guides/user/chapters/configuring.rst

```diff
@@ -170,6 +170,13 @@ Example using the configuration file:
   [foo]
   bar = 'hello world'
 
+Here you can also use without the quotes. For instance:
```

Comment:
Maybe better: "String and all following types could be used with or without quotes but using quotes for strings is important on the command line to safely handle empty spaces and distinguish it from a list type. Therefore, the following example will also be well handled:"

Reply:
@pevogam much better. Thanks.